### ISSUE TITLE
Is this repo still maintained?

### ISSUE THREAD

**Issue by metbril** [EBOOZ/TeamsStatus#40]
There are some pull requests from many months ago, with some nice fixes and features. I'm wondering if the repo is still maintained?

**Comment by metbril** [comment]
I am thinking of forking the repo and adding al those good PR's into it. And then keep that actively maintained. Anyone has any ideas on this? cc @vmax77

**Comment by chriscolden** [comment]
i have one here which has a bunch of PRs merged, bit of clean up and support for both running as a service or via a scheduled task. I'm not maintaining but if you just wanted something to grab quickly until this is updated, go for it.

https://github.com/chriscolden/TeamsStatus

**Comment by AntoineGS** [comment]
I've created a copy of the repo (with a credit section for the author here in the README) and will be maintaining it, PRs are welcomed:
https://github.com/AntoineGS/TeamsStatusV2
I will work through merging the PRs locally and pushing them making sure to keep the authors tied to the commits.

**Comment by mhoogenbosch** [comment]
> I've created a copy of the repo (with a credit section for the author here in the README) and will be maintaining it, PRs are welcomed: https://github.com/AntoineGS/TeamsStatusV2 I will work through merging the PRs locally and pushing them making sure to keep the authors tied to the commits.

Just installed your version, thanks a lot! Do you think you will be able to integrate the webcam PR? Would love that! Didn't get it to work on my cut paste actions.

**Comment by mhoogenbosch** [comment]
> I've created a copy of the repo (with a credit section for the author here in the README) and will be maintaining it, PRs are welcomed: https://github.com/AntoineGS/TeamsStatusV2 I will work through merging the PRs locally and pushing them making sure to keep the authors tied to the commits.

With your repro the settings file contains:

```
# Set entities to post to
$entityStatus = "sensor.teams_status"
$entityStatusName = "Microsoft Teams status"
$entityActivity = "sensor.teams_activity"
$entityActivityName = "Microsoft Teams activity"
$entityHeartbeat = "binary_sensor.teams_monitoring"
$entityHeartbeatName = "Microsoft Teams monitoring"
```

Which imiplies there is a Heartbeat binary sensor. I cannot find a pointer in the readme to create this. Is this just a binary sensor called 'teams monitoring'?

**Comment by AntoineGS** [comment]
Yeah I will review the camera monitoring and see about adding it.
Right now I am working through the issues with running it as a service though my next block of time to work on this might only be tomorrow evening.

For the Heartbeat sensor, this was from the original source, and it does seem to do absolutely nothing so I will remove it.

**Comment by mhoogenbosch** [comment]
> Yeah I will review the camera monitoring and see about adding it. Right now I am working through the issues with running it as a service though my next block of time to work on this might only be tomorrow evening.
> 
> For the Heartbeat sensor, this was from the original source, and it does seem to do absolutely nothing so I will remove it.

No worries, take your time. Just FYI, it works fine for me as a service. Main issue why it wasn't working for me was the running user. I have now changed it from local system to my own username and it runs fine.

**Comment by AntoineGS** [comment]
After reading up on the other conversations it makes sense that this would fix it, since it was the %appdata% that was yielding the inccorect path.
Thanks for the pointer!

**Comment by AntoineGS** [comment]
Alright I think I've got the camera working fairly well, it's not ideal since it does not seem to generate a log entry when it comes on/goes off so another entry needs to trigger the service to check the registry.
However after hanging up I seem to always get a few lines of log and it picks up the camera change.

I also ended up with your approach, the installer now asks for the windows password to install it under the logged-in user's account.
It assumes a local account for now, if someone ever needs it to work for non-local accounts or under a different account than the one logged in, I will add a configuration for it.

If you have any issues (or improvements to be made) you are welcome to open an issue or a PR on the new repo page!
